source/CommonJobs/CommonJobs.Application.EvalForm/DTOs/CalificationsEvaluationDto.cs

```diff
@@ -35,7 +35,9 @@ public class CalificationsEvaluationDto
 
         public List<string> Evaluators { get; set; }
 
-        public static CalificationsEvaluationDto Create(EmployeeEvaluation evaluation, List<string> evaluators, string currentPosition, string seniority)
+        public bool IsCompanyEvaluationDone { get; set; }
```

Comment:
So, ReadyToDevolution name is wrong because it does not match to the ReadyToDevolution state, right?

Do you think that it could be a good idea to rename these fields?